docker-compose.yml.v0.10

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
```

Comment:
We can delete this file as API components dont need Kafka v0.10 on docker compose